pages/_app.js

```diff
@@ -60,6 +61,7 @@ export default function App({ Component, pageProps }) {
         onGameUpdate={handleGameUpdate}
         onSetIsDead={handleSetIsDead}
       />
+      <AudioInterface></AudioInterface>
```

Comment:
I think this can be self closing. 

```suggestion
      <AudioInterface />
```